src/main/java/stringcalculator/Number.java

```diff
@@ -0,0 +1,41 @@
+package stringcalculator;
+
+/**
+ * 계산에 사용되는 숫자.
+ */
+public class Number {
+  private final Integer item;
+  
+  /**
+   * 계산에 사용되는 숫자에대해 유효성검사를 하여 통과시 해당값을 저장.
+   *
+   * @param value 문자형으로 표현된 숫자
+   */
+  public Number(String value) {
+    if (isInvalid(value)) {
+      throw new RuntimeException("유요하지 않은 계산 숫자입니다.");
+    }
```

Comment:
```suggestion
    validate(value);
```
를 하고 validate 함수 내부에서 예외를 처리하는건 어떨까요?

Reply:
유효성검사하는 함수내부에서 모든 예외처리하면 깔끔한데
유효성 검사를 호출하는 곳에서
"유효성 검사가 유효하지 않으니 다음 로직은 실행 할 수 없음"라고 인식할 수 있게 한 로직입니다.

보통은
로직 제어를 중점으로 표현 vs 로직 개념을 중점으로 표현
위 둘 중에 어떤것을 더 선호될까요??

Reply:
저는 로직 제어 중점 표현 vs 로직 개념 중점 표현 이라는 접근 보다는 읽기 쉬운 코드를 선호합니다.

생성자 내부에서는 값의 검증, 값의 할당을 할텐데, 해야할 일을 검증과 할당 두가지로 나누고 
검증과 관련된 모든 것을 하나의 함수로 묶어서 보다 간결하고 읽기 편하게 하는걸 선호해요.

또, 다음 로직을 수행할 수 없음을 굳이 표현하지 않더라도, 함수를 따라가다 보면 알 수 있는 부분이기도 하구요